src/controller/java/AndroidDeviceControllerWrapper.cpp

```diff
@@ -237,6 +241,9 @@ AndroidDeviceControllerWrapper * AndroidDeviceControllerWrapper::AllocateNew(
         setupParams.operationalKeypair                   = &ephemeralKey;
         setupParams.hasExternallyOwnedOperationalKeypair = false;
 
+        // TODO: make this conditional based upon state passed from java
+        opCredsIssuer->SetUseJavaCallbackForNOCRequest(true);
```

Comment:
We cannot force clients to use a way they have not used before. This breaks the previous API contract

Reply:
Removed this in favor of setting this configuration implicitly when a NOCChainIssuer is set on the java ChipDeviceController